src/store/flow/block-types/BaseBlock.ts

```diff
@@ -27,13 +26,9 @@ export const actions = {
       label: '',
       semantic_label: '',
       config: {},
-      exits: props?.exits ?? [
-        await dispatch('flow/block_createBlockDefaultExitWith', {
-          props: ({
-            uuid: await (new IdGeneratorUuidV4()).generate(),
-          }) as IBlockExit,
-        }, {root: true}),
-      ],
+      exits: props?.exits ?? await dispatch('flow/block_generateExitsBasedOnUiConfig', {
```

Comment:
I removed all exit generation within each block's `creatWith` in favour of this new line.